wrappers/swift/SDWLogger.swift

```diff
@@ -0,0 +1,237 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Errors thrown by Logger.
+enum SDKError: Error {
+    case ONEDSSDKError(reason: String)
+}
+
+/// Wrapper class around ObjC Logger class `ODWLogger` used to events.
+public class Logger {
+    private var odwLogger: ODWLogger;
+
+    /// Constructs `Logger` with `ODWLogger` object which would be wrapped around.
+    public init(logger: ODWLogger) {
```

Comment:
Could this not be public?, to avoid passing Obj-C wrapper object, breaking abstraction

Reply:
ODWLogger is being init with cpp class logger.
-(instancetype)initWithILogger:(ILogger*)logger

So construction of this class needs ODWLogger (or Ilogger).